setup.py

```diff
@@ -46,19 +46,19 @@ def find_version(*file_paths):
     'colorama==0.4.*',
     'graphene>=2.1,<3',
     'metomi-isodatetime==1!2.0.*',
-    'jinja2>=2.10.1, <2.11.0',
+    'jinja2==2.11.*',
```

Comment:
Alternatively, this could be

- `jinja2>=2.10.1, <2.12.0` (so users could still use 2.10.1 if they prefer?)
- `jinja2~=2.11.1` (gives users 2.11.0, or anything after that)